src/analyserbeats.cpp

```diff
@@ -33,14 +33,16 @@ AnalyserBeats::AnalyserBeats(ConfigObject<ConfigValue>* pConfig)
 AnalyserBeats::~AnalyserBeats() {
 }
 
-bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate, int totalSamples) {
+bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate,
+                               int totalSamples) {
     if (totalSamples == 0) {
         return false;
     }
 
     bool bPreferencesBeatDetectionEnabled = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_DETECTION_ENABLED)).toInt());
+            m_pConfig->getValueString(
+                             ConfigKey(BPM_CONFIG_KEY, BPM_DETECTION_ENABLED))
+                    .toInt());
```

Comment:
Given that choice, I agree with its decision because this is clearer than that would've been. (Though the indentation would have probably lined up better.)